buildlib/azure-pipelines-pr.yml

```diff
@@ -76,6 +79,25 @@ stages:
 
   - stage: Build
     jobs:
+      - job: build_source
+        pool:
+          name: MLNX
+          demands:
+            - ucx_instinct -equals yes
+        strategy:
+          matrix:
+            centos7_cuda11_1:
+              CONTAINER: centos7_cuda11_1
+              CONFIGURE_OPTS:
+        container: $[ variables['CONTAINER'] ]
+
+        steps:
+          - checkout: self
+            clean: true
+
+          - bash: ./autogen.sh
+            displayName: Setup autotools
```

Comment:
this step can be bigger, maybe use additional yml file?

Reply:
it just for debug, will launch like in jenkins via test_jenkins.sh (it've done)